mailpoet/mailpoet
# Add new Amazon SES locations - [MAILPOET-4041]

[MAILPOET-4041](https://mailpoet.atlassian.net/browse/MAILPOET-4041)

Testing Instructions:
1. Go to MailPoet > Settings > Send With tab
2. Click to Configure link
3. Then Select AmazonSES as the sending option
4. Click to see the whole list of locations inder Region
5. Compare all locations with [this list](https://docs.aws.amazon.com/general/latest/gr/rande.html#ses_region)
6. Click Cancel at the end so that it remains with MSS service
